tests/expectations/tests/libclang-3.9/issue-1198-alias-rust-enum.rs

```diff
@@ -0,0 +1,18 @@
+/* automatically generated by rust-bindgen */
+
+#![allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals)]
```

Comment:
Looks like CI is not really happy about this (needs expectations for other libclang versions). But does this need to be really a libclang-3.9+ test?